timespec/timespec_test.go

```diff
@@ -24,6 +24,36 @@ var _ = Describe("Timespec", func() {
 			Ω(spec.String()).Should(Equal("hourly at 45 after"))
 		})
 
+		It("can stringify cardinal hourly specs", func() {
+			spec := &Spec{
+				Interval:    Hourly,
+				TimeOfDay:   inMinutes(4, 39),
+				Cardinality: 6,
+			}
+
+			Ω(spec.String()).Should(Equal("every 6 hours from 4:39"))
+		})
+
+		It("can stringify half hour specs", func() {
+			spec := &Spec{
+				Interval:    Hourly,
+				TimeOfDay:   inMinutes(0, 15),
+				Cardinality: 0.5,
+			}
+
+			Ω(spec.String()).Should(Equal("every 0.50 hours from 0:15"))
+		})
+
+		It("can stringify quarter hour specs", func() {
+			spec := &Spec{
+				Interval:    Hourly,
+				TimeOfDay:   inMinutes(0, 15),
+				Cardinality: 0.25,
+			}
+
+			Ω(spec.String()).Should(Equal("every 0.25 hours from 0:15"))
```

Comment:
Likewise, this should return "every quarter hour from 0:15", so that we don't generate a string we cannot parse.